website/docs/artist_getting_started.md

```diff
@@ -11,7 +11,12 @@ import TabItem from '@theme/TabItem';
 
 AYON comes in packages for Windows (10 or Server), Mac OS X (Mojave or higher), and Linux distribution (Centos, Ubuntu).
 
-Your studio administrator should provide you with an installation file. If you are testing or working by yourself, you can also grab installation files from the releases; https://github.com/ynput/OpenPype/releases
+In AYON you can download the installer from within the website:
+![A screenshot of the AYON main menu with the "Download Launcher" menu expanded](assets/ayon_download_installer.png)
+
+A "Download Launcher" blue button should be present, which provides a binary for the Operating System you are accessing the webistes, alternatively you can download a specific binary from the dropdown in the main menu.
+
+If you are testing or working by yourself, you can also grab installation files from the releases: https://github.com/ynput/ayon-launcher/releases
```

Comment:
Opinion on this? I don't like the link explicit like this and ending the sentence without the dot 😅
```suggestion
If you are testing or working by yourself, you can also grab installation files from the [releases](https://github.com/ynput/ayon-launcher/releases).
```